terraform/locals.tf

```diff
@@ -0,0 +1,129 @@
+locals {
+  project         = "trade-tariff"
+  no_reply        = "no-reply@${local.base_url}"
+  base_url        = "${local.project}.service.gov.uk"
+  environment_key = var.environment == "development" ? "development" : var.environment
+}
+
+locals {
+
+  backend_common_vars = [
+    {
+      name  = "ALLOW_MISSING_MIGRATION_FILES"
+      value = "true"
+    },
+    {
+      name  = "AWS_BUCKET_NAME"
+      value = "trade-tariff-persistence-${var.environment}"
+    },
+    {
+      name  = "BETA_SEARCH_MAX_HITS"
+      value = "1000"
+    },
+    {
+      name  = "DB_POOL"
+      value = "20"
+    },
+    {
+      name  = "FRONTEND_HOST"
+      value = "https://${local.environment_key}.${local.base_url}/"
+      # TODO: scope this due to www
+    },
+    {
+      name  = "MALLOC_ARENA_MAX"
+      value = "2"
+    },
+    {
+      name  = "MAX_THREADS"
+      value = "6"
+    },
+    {
+      name  = "NEW_RELIC_AGENT_ENABLED"
+      value = "false"
+    },
+    {
+      name  = "NEW_RELIC_APP_NAME"
+      value = "tariff-uk-backend-${var.environment}"
+    },
+    {
+      name  = "NEW_RELIC_DISTRIBUTED_TRACING"
+      value = "true"
+    },
+    {
+      name  = "PAAS_S3_SERVICE_NAME"
+      value = "tariff-pdf-${local.environment_key}"
+    },
+    {
+      name  = "RACK_TIMEOUT_SERVICE"
+      value = "50"
+    },
+    {
+      name  = "RACK_TIMEOUT_SERVICE_TIMEOUT"
+      value = "50"
+    },
+    {
+      name  = "RACK_TIMEOUT_WAIT_TIMEOUT"
+      value = "100"
+    },
+    {
+      name  = "SENTRY_DSN"
+      value = var.tariff_backend_sentry_dsn
+    },
+    {
+      name  = "SENTRY_PROJECT"
+      value = "tariff-backend"
+    },
+    {
+      name  = "SPELLING_CORRECTOR_BUCKET_NAME"
+      value = "trade-tariff-search-configuration-${var.environment}"
```

Comment:
Looking at [`s3.tf`](https://github.com/trade-tariff/trade-tariff-platform-aws-terraform/blob/da794a99246fb1782b330affb9be620fe96b3198/environments/common/s3/s3.tf#L2), this bucket doesn't exist in the map. Can we get a PR raised to add this one - and change the resource name here to match the standard